<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light SemiCondensed</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Facul-Tech</string>
  </property>
  <property name="styleSheet">
   <string notr="true">  background-color: #4070f4;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">margin: 5px;
font: 13pt &quot;Bahnschrift&quot;;</string>
        </property>
        <property name="text">
         <string>Locales</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 5px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Nombre del Local</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 30px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Docente</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">margin:10px 10px 5px 10px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Tipo de Local</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 5px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Nombre del Responsable</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="txt_nombre_local">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 5 5 5;
outline:none;
border:1px solid #aaa;
padding:7px;
border-radius: 5px;
color: #333;
}

QLineEdit:focus{
border: 1px solid #007acc;

}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese nombre del Local</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="txt_responsable">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 5 5 5;
outline:none;
border:1px solid #aaa;
padding:7px;
border-radius: 5px;
color: #333;
}

QLineEdit:focus{
border: 1px solid #007acc;

}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese nombre del Responsable</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QRadioButton" name="rbtn_no">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 5px 5px 5px 30px;

</string>
        </property>
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="QTableWidget" name="tabla_locales">
        <property name="styleSheet">
         <string notr="true">QHeaderView::section{
font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
}

QTableWidget QtableCornerButton::section{
font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
border: solid 1px black;




}
QTableWidget{font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
border:solid 1px gray;
margin: 30px 0 0 0;}
</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="rbtn_oficina">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 0 5px 0 10px;
</string>
        </property>
        <property name="text">
         <string>Oficina</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rbtn_aula">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 0 5px 0 10px;

</string>
        </property>
        <property name="text">
         <string>Aula</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="rbtn_departamento">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 0 5px 0 10px;


</string>
        </property>
        <property name="text">
         <string>Departamento</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="rbtn_si">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 0 5px 0px 30px;

</string>
        </property>
        <property name="text">
         <string>Si</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="rbtn_laboratorio">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin: 0 5px 0 10px;
</string>
        </property>
        <property name="text">
         <string>Laboratorio</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_insertar">
           <property name="font">
            <font>
             <family>Bahnschrift Light SemiCondensed</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
  background-color: #4070f4;
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: darkblue;}</string>
           </property>
           <property name="text">
            <string>Insertar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_actualizar">
           <property name="styleSheet">
            <string notr="true">QPushButton{ 
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
  background-color: #4070f4;
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: darkblue;}</string>
           </property>
           <property name="text">
            <string>Actualizar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_eliminar">
           <property name="styleSheet">
            <string notr="true">QPushButton{ 
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
  background-color: #4070f4;
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: darkblue;}</string>
           </property>
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cerrar">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  text-align: center;
  line-height: 18px;
  background-color: #4070f4;
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: darkblue;}</string>
           </property>
           <property name="text">
            <string>Cerrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
